### PR TITLE
Create API blogpost from budget request text

### PR DESCRIPTION
**Pull request by brigsz**
This is a post from the budget request for Web Services/Web API. It needs formatting a bit probably

### CODE REVIEW

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+As agencies continually move toward data-driven approaches to serve the citizens of Utah, location data must be consistently robust, repeatable, and easily accessed by problem solvers so they can make the most relevant data-driven decisions. To meet the increasing demand for consistent statewide data, UGRC needs consistent, increased funding. This increased funding would support several web services, including Next Generation 911 (NG911), lieutenant governor voting efforts and administrative boundary certification, Medicaid fraud detection, treasurer’s office unclaimed property section, fleet management, streamlined sales tax, and other programs that require a citizen location to provide the appropriate services.
+
+
+Why UGRC Web Services?
```
Comment (steveoh) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:25]:
```suggestion
Why UGRC Web Services? 

```

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Voting Applications: Assignment of voters to precincts, and precincts to districts 
+
+Detection of Medicaid and other types of fraud
+
+Assistance with the Watershed Restoration Initiative 
+
+Additional web services funding will allow UGRC to make required changes and improvements that are needed in order to handle the widespread increase in demand for these centralized UGRC services. 
+
+UGRC has used existing resources to develop web services to their current state and have tried to continue on this path; however, the significant adoption and reliance on the UGRC Web API requires a much more focused and attentive approach, as monthly requests to the API have risen in excess of twelve million. UGRC’s need to generate revenue by providing cost-recovery project work greatly limits the time we are able to put toward the Web API. We are simply not able to maintain it like we should, such as updating dependencies and implementing new technologies to ensure its robustness. These standard IT practices should be happening regularly; however, the last major update to this enterprise service was over five years ago.
```
Comment (steveoh) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:38]: **thought**: i think we need to be careful with our wording here so we don't make people question the accuracy etc of the service.

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Benefits and Value
```
Comment (steveoh) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:40]:
```suggestion
Benefits and Value

```

**Review by steveoh — COMMENTED** [review]
a quick once over. Ditch the double spaces 😆 and a few other suggestions

File: _posts/2022-03-21-ugrc-web-services-api-funding.md (hunk 2 above)
Comment (brigsz) [reply _posts/2022-03-21-ugrc-web-services-api-funding.md:38]: I will try to reword this

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+title: 'UGRC web services API funding'
+author:
+  display_name: Matt Peters
+  email: mpeters@utah.gov
+date: 2022-03-21 14:10:46
+categories: []
```
Comment (jacobdadams) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:7]: The post isn't showing up in the recent posts area in the site header. Do we need a tag for that to happen?

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+As agencies continually move toward data-driven approaches to serve the citizens of Utah, location data must be consistently robust, repeatable, and easily accessed by problem solvers so they can make the most relevant data-driven decisions. To meet the increasing demand for consistent statewide data, UGRC needs consistent, increased funding. This increased funding would support several web services, including Next Generation 911 (NG911), lieutenant governor voting efforts and administrative boundary certification, Medicaid fraud detection, treasurer’s office unclaimed property section, fleet management, streamlined sales tax, and other programs that require a citizen location to provide the appropriate services.
```
Comment (jacobdadams) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:23]:
```suggestion
As agencies continually move toward data-driven approaches to serve the citizens of Utah, location data must be consistently robust, repeatable, and easily accessed by problem solvers so they can make the most relevant data-driven decisions. To meet the increasing demand for consistent statewide data, UGRC needs consistent, increased funding. This increased funding would support several web services, including Next Generation 911 (NG911), lieutenant governor voting efforts and administrative boundary certification, Medicaid fraud detection, treasurer’s office unclaimed property section, fleet management, streamlined sales tax, and other programs that require a citizen's address location to provide the appropriate services.
```

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+Funding for UGRC web services will also add value in the following areas: 
+
+- Promote economic advancement - Citizen addresses paired with broadband availability areas inform where service is needed. The Broadband Economic Development Map shows other data that helps ascertain the location of economic opportunity in rural Utah.
+- Assist with education innovation and investment - When students move from one district to another, geocoding helps identify the to and from district, then funds can be transferred.
+- Streamline and modernize state government - UGRC web services offer an authoritative data source, a single source of truth for all agencies providing services based on the location of citizens.
```
Comment (jacobdadams) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:48]:
```suggestion
- Streamline and modernize state government - UGRC web services offer an authoritative data source, a single source of truth for all agencies providing services based on the location of citizens' addresses.
```

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+All Utahns will ultimately benefit from UGRC web services, as locational accuracy of citizens and their relationship to government services and opportunities affects many aspects of the quality of citizens’ lives from safety (911), education (school districts), voting and elections (districts and precincts), monetary (tax entities and areas), recreation (proximity to parks and outdoor areas), and health care (available services and providers).
```
Comment (jacobdadams) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:50]:
```suggestion
All Utahns will ultimately benefit from UGRC web services, as locational accuracy of citizens' addresses and their relationship to government services and opportunities affects many aspects of the quality of citizens’ lives from safety (911), education (school districts), voting and elections (districts and precincts), monetary (tax entities and areas), recreation (proximity to parks and outdoor areas), and health care (available services and providers).  
```

**Review by jacobdadams — COMMENTED** [review]
It may be good to be explicit that we're talking about address locations not tracking citizens' every move. We know that's the case, but we've also seen where our words are taken out of context for interesting purposes.

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+  email: mpeters@utah.gov
+date: 2022-03-21 14:10:46
+categories: []
+tags:
+  - sgid
+  - web api
+  - ugrc funding
+---
+
+UGRC received additional funds for our work on the Web API and associated services. Below is the justification I submitted.
```
Comment (nathankota) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:14]:
```suggestion
UGRC received additional funds for our work on the Web API and associated services. Below is the justification I submitted for the request to the legislature.
```

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+## UGRC Web Services Enhancements Budget Request
+{: .text-left}
+
+The Utah Geospatial Resource Center (UGRC) is responsible for acquiring, aggregating, and developing geospatial data for the State of Utah. Over the years UGRC has worked with countless partners to amass important spatial data, which is stored in the State Geographic Information Datasource (SGID). UGRC web services, specifically the web application programming interface (API), allow users to visualize and use this SGID data in a myriad of ways, from anywhere, for whatever reason, without licensed software. 
+
+In the early days of UGRC, the Web API was not a known technology; now it has become the de facto standard for enhancing and providing access to the SGID data. UGRC does not have a current budget for this effort. For years UGRC developed the technology with funds from: other projects that utilize the API, appropriation funding, and small enhancement request funding from other agencies and organizations that are interested in supporting the API. However, none of these undedicated sources are sufficient to properly enhance, update, and maintain the UGRC Web API system.
```
Comment (nathankota) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:21]:
```suggestion
In the early days of UGRC, the Web API was not a known technology; now it has become the de facto standard for enhancing and providing access to the SGID data. UGRC does not have a current budget for this effort. For years UGRC developed the technology with funds from other projects that utilize the API, appropriation funding, and small enhancement request funding from other agencies and organizations that are interested in supporting the API. However, none of these undedicated sources are sufficient to properly enhance, update, and maintain the UGRC Web API system.
```

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+{: .text-left}
+
+UGRC web services require funding because they support so many other government applications, including the following services:
+
+- NG911: Improved routing of emergency calls to a public safety answering point and dispatch of emergency vehicles
+- Voting Applications: Assignment of voters to precincts, and precincts to districts 
+- Detection of Medicaid and other types of fraud
+- Assistance with the Watershed Restoration Initiative 
+
+Additional web services funding will allow UGRC to make required changes and improvements that are needed in order to handle the widespread increase in demand for these centralized UGRC services.
```
Comment (nathankota) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:35]:
```suggestion
Additional web services funding will allow UGRC to make required changes and improvements that are needed to handle the widespread increase in demand for these centralized UGRC services. 
```

File: _posts/2022-03-21-ugrc-web-services-api-funding.md (hunk 4 above)
Comment (steveoh) [reply _posts/2022-03-21-ugrc-web-services-api-funding.md:7]: Featured adds that

**Review by nathankota — COMMENTED** [review]
I added a few frivolous edits; otherwise, it looks good to me.

**Review by brigsz — COMMENTED** [review]
okay, I have implemented the changes suggested.  I am a little unsure about the tag Jake speaks of.  The article does not necessarily need to be in the featured category

**Comment by jacobdadams** [comment]
> okay, I have implemented the changes suggested. I am a little unsure about the tag Jake speaks of. The article does not necessarily need to be in the featured category

I was just curious why it wasn't showing up in the header list.

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+{: .text-left}
+
+UGRC web services require funding because they support so many other government applications, including the following services:
+
+- NG911: Improved routing of emergency calls to a public safety answering point and dispatch of emergency vehicles
+- Voting Applications: Assignment of voters to precincts, and precincts to districts 
+- Detection of Medicaid and other types of fraud
+- Assistance with the Watershed Restoration Initiative 
+
+Additional web services funding will allow UGRC to make required changes and improvements that are needed to handle the widespread increase in demand for these centralized UGRC services.
```
Comment (steveoh) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:35]:
```suggestion
Additional web services funding will allow UGRC to raise the bar for making location data available. This is especially relevant with the widespread increase in demand for these centralized and authoritative UGRC services. 
```

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+- Voting Applications: Assignment of voters to precincts, and precincts to districts 
+- Detection of Medicaid and other types of fraud
+- Assistance with the Watershed Restoration Initiative 
+
+Additional web services funding will allow UGRC to make required changes and improvements that are needed to handle the widespread increase in demand for these centralized UGRC services. 
+
+UGRC has used existing resources to develop web services to their current state and have tried to continue on this path; however, the significant adoption and reliance on the UGRC Web API requires a much more focused and attentive approach, as monthly requests to the API have risen in excess of twelve million. UGRC’s need to generate revenue by providing cost-recovery project work greatly limits the time we are able to put toward the Web API. We are simply not able to maintain it like we should, such as updating dependencies and implementing new technologies to ensure its robustness. These standard IT practices should be happening regularly; however UGRC always makes sure the system is viable, reliable, and a valuable resource.
```
Comment (steveoh) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:37]:
```suggestion
UGRC has used existing resources to develop web services to their current state and have tried to continue on this path; however, the significant adoption and reliance on the UGRC Web API requires a much more focused and attentive approach, as monthly requests to the API have risen in excess of twelve million. UGRC’s need to generate revenue by providing cost-recovery project work greatly limits the time we are able to put toward the Web API. We are simply not able to maintain and enhance the API as we would like to; however UGRC always makes sure the system is viable, reliable, and a valuable resource.
```

**Review by steveoh — COMMENTED** [review]
unnecessary but i think beneficial edit suggestions.

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+
+As agencies continually move toward data-driven approaches to serve the citizens of Utah, location data must be consistently robust, repeatable, and easily accessed by problem solvers so they can make the most relevant data-driven decisions. To meet the increasing demand for consistent statewide data, UGRC needs consistent, increased funding. This increased funding would support several web services, including Next Generation 911 (NG911), lieutenant governor voting efforts and administrative boundary certification, Medicaid fraud detection, treasurer’s office unclaimed property section, fleet management, streamlined sales tax, and other programs that require a citizen's address location to provide the appropriate services.
```
[the remark this answers is not in the document]
Comment (brigsz) [reply _posts/2022-03-21-ugrc-web-services-api-funding.md]: Mmmmm, yah, that does not read correctly

[the remark this answers is not in the document]
Comment (brigsz) [reply _posts/2022-03-21-ugrc-web-services-api-funding.md]: maybe support several uses of web services,      Or something of that sort

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 14]
```diff
@@ -0,0 +1,50 @@
+---
+title: 'UGRC Web Service Funding'
```
Comment (steveoh) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:2]:
```suggestion
title: 'UGRC Web Service Legislative Funding Request Update'
```

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 15]
```diff
@@ -0,0 +1,50 @@
...
+
+## Benefits and Value
+{: .text-left}
+
+Improving UGRC web services is at the crux of modernization and innovation efforts for Utah state government. By continually developing, maintaining, and improving the quality and accessibility of citizen address coordinates and related spatial data, the Web API provides a single, consistent source of truth from which all government agencies can pull data about the relationship of their services to the citizens of Utah. With increased funding, UGRC web services will be ready to handle the increasing demand for agencies to integrate spatial data into their business processes. 
+
+The Funding UGRC has received for web services will also add value in the following areas:
```
Comment (steveoh) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:44]: Stick to past tense

```suggestion
The Funding UGRC is requesting for web services will also add value in the following areas: 
```

**Review by steveoh — COMMENTED** [review]
ok i think i'm done after this one :)

File: _posts/2022-03-21-ugrc-web-services-api-funding.md [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+  email: mpeters@utah.gov
+date: 2022-03-21 14:10:46
+categories: []
+tags:
+  - sgid
+  - web api
+  - ugrc funding
+---
+
+UGRC received additional funds for our work on the Web API and associated services. UGRC did not receive all funds requested however we did get ongoing funds, so users can expect to see enhancements in the near future.  UGRC is also very open to suggestions for changes to our web services. Below is the justification I submitted for the request to the legislature.
```
Comment (steveoh) [inline _posts/2022-03-21-ugrc-web-services-api-funding.md:14]:
```suggestion
UGRC received additional funds for our work on the Web API and associated services. UGRC did not receive all funds requested however we did get ongoing funds, so users can expect to see enhancements in the near future. UGRC is also very open to suggestions for changes to our web services. Below is the justification I submitted for the request to the legislature.
```

**Review by steveoh — DISMISSED** [review]

**Review by steveoh — APPROVED** [review]

**Pull request merged**